src/libmain/common-args.cc

```diff
@@ -71,13 +71,8 @@ MixCommonArgs::MixCommonArgs(const std::string & programName)
         }}
     });
 
-    std::string cat = "Options to override configuration settings";
-    globalConfig.convertToArgs(*this, cat);
```

Comment:
I think you made it not have those flags entirely.

Reply:
Ah yes... It probably needs to be sensible about the hidden property at the JSON output.